src/mapgen/mapgen_v7.cpp

```diff
@@ -131,27 +146,27 @@ MapgenV7::~MapgenV7()
 
 
 MapgenV7Params::MapgenV7Params():
-	np_terrain_base      (4,    70,   v3f(600,  600,  600),  82341, 5, 0.6,  2.0),
-	np_terrain_alt       (4,    25,   v3f(600,  600,  600),  5934,  5, 0.6,  2.0),
-	np_terrain_persist   (0.6,  0.1,  v3f(2000, 2000, 2000), 539,   3, 0.6,  2.0),
-	np_height_select     (-8,   16,   v3f(500,  500,  500),  4213,  6, 0.7,  2.0),
-	np_filler_depth      (0,    1.2,  v3f(150,  150,  150),  261,   3, 0.7,  2.0),
-	np_mount_height      (256,  112,  v3f(1000, 1000, 1000), 72449, 3, 0.6,  2.0),
-	np_ridge_uwater      (0,    1,    v3f(1000, 1000, 1000), 85039, 5, 0.6,  2.0),
-	np_floatland_base    (-0.6, 1.5,  v3f(600,  600,  600),  114,   5, 0.6,  2.0),
-	np_float_base_height (48,   24,   v3f(300,  300,  300),  907,   4, 0.7,  2.0),
-	np_mountain          (-0.6, 1,    v3f(250,  350,  250),  5333,  5, 0.63, 2.0),
-	np_ridge             (0,    1,    v3f(100,  100,  100),  6467,  4, 0.75, 2.0),
-	np_cavern            (0,    1,    v3f(384,  128,  384),  723,   5, 0.63, 2.0),
-	np_cave1             (0,    12,   v3f(61,   61,   61),   52534, 3, 0.5,  2.0),
-	np_cave2             (0,    12,   v3f(67,   67,   67),   10325, 3, 0.5,  2.0)
+	np_terrain_base      (4.0,   70.0,  v3f(600,  600,  600),  82341, 5, 0.6,  2.0),
```

Comment:
note the v3f initialization values are double converted to floats, please fix it

Reply:
Ok. `v3f(600.0f,  600.0f,  600.0f)` or `v3f(600.0,  600.0,  600.0)`?